ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -310,7 +310,7 @@ public static ByteBuf readNettyByteBufFromCrcInputStream(CrcInputStream crcStrea
    * @param <T> The exception to throw in this operation.
    */
   @FunctionalInterface
-  public static interface ByteBufferFunction<T extends Throwable> {
+  public interface ByteBufferFunction<T extends Throwable> {
```

Comment:
It was a suggested change from intellij. The static is redundant since inner interfaces are inherently static (no outer class instance variables can be accessed)